### PR TITLE
Article about zone files

### LINKED ISSUE #10: Article about importing records
Using:

- Domain zone file
- API ?

### PR DESCRIPTION
**Pull request by jacegu**
Transforms the article about getting a zone file to one that talks about:

- What a zone file is
- How to import records using a zone file
- How to export a domain's zone to a zone file

Closes #10

### CODE REVIEW

File: content/articles/zone-files.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Domain zone files
+excerpt: Welcome to DNSimple. This page is about zone files. Hosted DNS has never been this easy.
```
Comment (weppos) [inline content/articles/zone-files.markdown]: I vote to change this excerpt into something more inline with the content.

> This article explains what is a DNS zone file and how to import/export zone text files in DNSimple.

**Comment by weppos** [comment]
Excellent article! Does it make sense to mention the text file format is essentially the BIND format?

**Comment by jacegu** [comment]
@weppos Thanks! :smile: 

What do you mean by _essentially_ the BIND format? I thought it was **in** the BIND format. 

There is [a note about it](https://github.com/aetrion/dnsimple-support/commit/88d13fdc4297319b77b4ef83be0ed84a5c4679c4#diff-f6b6d87a08a7792f6260873f60e364acR15) in the first paragraph. Maybe it has to be more clear?

File: content/articles/zone-files.markdown (hunk 1 above)
Comment (jacegu) [inline content/articles/zone-files.markdown]: :+1: 

Thought we had to leave the :hankey: for SEO purposes. Happy to change it :wink: !

**Comment by weppos** [comment]
I missed the note in the first paragraph. This is exactly what I meant, I just didn't notice it was already there :wink: 

:shipit:

**Comment by sbastn** [comment]
Could it be useful to have a BIND file format? There is one in Wikipedia

**Pull request merged**